enterprise/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_3/runtime/PipelineInformation.scala

```diff
@@ -20,15 +20,113 @@
 
 package org.neo4j.cypher.internal.compatibility.v3_3.runtime
 
+import org.neo4j.cypher.internal.compiler.v3_3.planner.logical.plans.LogicalPlan
 import org.neo4j.cypher.internal.frontend.v3_3.InternalException
 import org.neo4j.cypher.internal.frontend.v3_3.symbols.CypherType
 
+import scala.collection.mutable
 
 object PipelineInformation {
   def empty = new PipelineInformation(Map.empty, 0, 0)
 
   def apply(slots: Map[String, Slot], numberOfLongs: Int, numberOfReferences: Int) =
     new PipelineInformation(slots, numberOfLongs, numberOfReferences)
+
+  def toString(startFrom: LogicalPlan, m: Map[LogicalPlan, PipelineInformation]): String = {
```

Comment:
I'm not seeing any calls to this new PipelineInformation.toString(....) method in the PR. Is this part of some other work?